old/build/BuildInfo.targets

```diff
@@ -9,7 +9,7 @@
     </GetCurrentRuntimeInformation>
```

Comment:
Are we keeping the "old/build" folder for a reason?

Reply:
Haven't deleted it yet as I took some stuff from it. No particular reason to keep it really, we just haven't needed to delete it yet.